### PR TITLE
created webscrapper

### PR DESCRIPTION
**Pull request by kbojarowska**

### CODE REVIEW

File: scrapper/src/main.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from bs4 import BeautifulSoup
+import json
+
+
+chrome_options = Options()
+chrome_options.add_argument("--headless")
+service = Service('/path/to/chromedriver')
```
Comment (dpionk) [inline scrapper/src/main.py:13]: I don't quite understand why  '/path/to/chromedriver' is provided here, it does not look like a valid path

File: package.json [hunk 2]
```diff
@@ -1,8 +0,0 @@
-{
```
Comment (dpionk) [inline package.json:1]: why delete package.json?

File: scrapper/src/main.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                'name': name,
+                'description': info,
+                'photo': image,
+                'ingredients': ingredients,
+                'recipe': recipe
+            }
+            data.append(recipe_data)
+
+json_data = json.dumps(data, indent=4)
+print(json_data)
```
Comment (dpionk) [inline scrapper/src/main.py:50]: unnecessary print

File: skin-resque-api/init-mongo.js [hunk 4]
```diff
@@ -0,0 +1,549 @@
+db = new Mongo().getDB("skin-resque");
+
+db.createCollection('Cosmetics', {capped: false});
+
+db.Cosmetics.insert([
```
Comment (dpionk) [inline skin-resque-api/init-mongo.js:5]: I think that web scraper should insert data to the database

File: package.json (hunk 2 above)
Comment (dpionk) [reply package.json:1]: I now know why xD

File: skin-resque-api/init-mongo.js (hunk 4 above)
Comment (kbojarowska) [reply skin-resque-api/init-mongo.js:5]: This is adding data by docker-compose

**Review by dpionk — APPROVED** [review]

**Pull request merged**